extensions/amp-script/0.1/amp-script.js

```diff
@@ -133,6 +136,25 @@ export class AmpScript extends AMP.BaseElement {
     });
   }
 
+  /**
+   * @override
+   */
+  onLayoutMeasure() {
```

Comment:
Nit: use `onMeasureChanged`. `onLayoutMeasure` can fire multiple times with the same measurements.

Reply:
will `onMeasureChanged` also fire on the first measurement?

Reply:
Yes.